charts/jindocache/templates/fuse/daemonset.yaml

```diff
@@ -0,0 +1,247 @@
+{{- $notEnableDnsConfig := (eq .Values.master.dnsServer "1.1.1.1") }}
+
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ template "jindofs.fullname" . }}-fuse
+  labels:
+    app: {{ template "jindofs.name" . }}
+    chart: {{ template "jindofs.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    role: jindofs-fuse
```

Comment:
Should the role name be jindocache-fuse?

Reply:
done